app/queries/movie_query.rb

```diff
@@ -0,0 +1,32 @@
+class MovieQuery
+  def initialize(params)
+    @params = params
+  end
+
+  def results
+    @results = Movie.all
+    actor_filter
+    order_results
+    @results
+  end
+
+  private
+
+  def order_results
+    sort = 'id'
+    direction = 'asc'
+    if @params[:sort].present?
+      direction = 'desc' if @params[:sort].start_with?('-')
+      sort = @params[:sort].delete_prefix('-')
+    end
+    raise Errors::InvalidSortParam unless Movie.attribute_names.include?(sort)
```

Comment:
There is explicit list of attributes in serializer, but here we can sort by attributes that are not visible in serializer.